FAQ.md

```diff
@@ -302,6 +303,37 @@ Set `DEVITO_IGNORE_UNKNOWN_PARAMS=1` to avoid Devito raising an exception if one
 [top](#Frequently-Asked-Questions)
 
 
+## What are the accepted combinations of PLATFORM, ARCH, and LANGUAGE
+
+#### LANGUAGE={C,openmp}
+
+These two languages can be used with virtually any PLATFORM and ARCH.
+
+With a device PLATFORM (e.g., `nvidiaX` or `amdgpuX`), the compiler will generate OpenMP code for device offloading.
+
+When using OpenMP offloading, it is however recommended to stick to the corresponding vendor compiler, so `ARCH=amdclang` for AMD, `ARCH={icc,icx,intel}` for Intel, and `ARCH=nvc` for NVidia.
+
+#### LANGUAGE=openacc
+
+Recommended: `PLATFORM=nvidiaX` and `ARCH=nvc`.
+
+The legacy PGI compiler is also supported via `ARCH=pgcc`.
+
+#### LANGUAGE=cuda
+
+Requires: `PLATFORM=nvidiaX` and `ARCH=cuda`.
+
+Also requires DevitoPRO.
+
+#### LANGUAGE=hip
+
+Requires: `PLATFORM=amdgpuX` and `ARCH=hip`.
+
+Also requires DevitoPRO.
+
+
+[top](#Frequently-Asked-Questions)
```

Comment:
Since this is here, you should drop line 303

https://github.com/devitocodes/devito/pull/2195/files#diff-c7bd425fd98aad1f9fef20099637bcbdcfadeb566ba1f83bb40ce484f195b8cfR303